.devcontainer/post-install.sh

```diff
@@ -7,4 +7,4 @@ apt update
 apt install -y mkdocs
 
 # Trust dotnet developer certs
-dotnet dev-certs https --check --trust
\ No newline at end of file
+dotnet dev-certs https --check --trust
```

Comment:
@linkdotnet FWIW, This line does work for me and err with "No valid certificates found"

Is it any required? What's the goal of it?

Reply:
If you wanna serve the samples (as they have https) the dotnet dev-certs have to be installed.
So strictly - no not necessary, but a nice goody.